docs/host/host.md

```diff
@@ -268,7 +268,13 @@ Before removing a disk, you must first evict Longhorn replicas on the disk.
 
 :::note
 
-The replica data would be rebuilt to another disk automatically to keep the high availability.
+The replica data would be rebuilt to another disk automatically to keep the high availability. There is a known issue on v2 where single replicated volume will be attempted
```

Comment:
is this expanded note dedicated for head branch, or is also fit for existing ones like v1.6, v1.5? when true, please  copy to them